client/src/components/RichTextEditor.js

```diff
@@ -291,8 +291,9 @@ const serialize = node => {
     case "block-quote":
       return `<blockquote>${children}</blockquote>`
     case ANET_LINK:
+      return `<a href="${getUrlFromEntityInfo(node)}">${node.entityType}</a>`
     case EXTERNAL_LINK:
-      return `<a href="${getUrlFromEntityInfo(node)}">${children}</a>`
+      return `<a href="${node.url}">${children}</a>`
```

Comment:
This line doesn't need to be changed, so let's keep it as:
```
      return `<a href="${getUrlFromEntityInfo(node)}">${children}</a>`
```
so we hide whatever `getUrlFromEntityInfo` does in this case from the caller.

Reply:
Applied